database/factories/OrderFactory.php

```diff
@@ -2,10 +2,27 @@
 
 namespace Database\Factories;
 
+use App\Models\User;
 use Illuminate\Database\Eloquent\Factories\Factory;
+use illuminate\Support\Collection;
 
 class OrderFactory extends Factory
 {
+    private $statuses;
+
+    public function __construct($count = null,
+        ?Collection $states = null,
+        ?Collection $has = null,
+        ?Collection $for = null,
+        ?Collection $afterMaking = null,
+        ?Collection $afterCreating = null,
+        $connection = null,
+        ?Collection $recycle = null)
+    {
+        parent::__construct($count, $states, $has, $for, $afterMaking, $afterCreating, $connection, $recycle);
+        $this->statuses = ["processing", "shipped", "delivered"];
```

Comment:
The statuses in this your array don't match the statuses in the [migration for the "order" table](https://github.com/Web-Ange/ecommerce-php/blob/cd61a3af9c7740d4e25804eafd9c334a337d09dd/database/migrations/2024_03_28_230537_create_order_table.php#L21). You need to capitalize them to `["Processing", "Shipped", "Delivered"]`.